nautobot/core/templates/generic/object_list.html

```diff
@@ -104,17 +104,39 @@
             </ul>
         </div>
     {% endif %}
-    {% if permissions.add and 'add' in action_buttons %}
-        {% add_button content_type.model_class|validated_viewname:"add" %}
-    {% endif %}
+    <div class="btn-group">
+        {% if permissions.add and 'add' in action_buttons %}
+            {% add_button content_type.model_class|validated_viewname:"add" %}
+            <button type="button" id="actions-dropdown" class="btn btn-primary dropdown-toggle" data-toggle="dropdown">
+                <span class="caret"></span>
+                <span class="sr-only">Toggle Dropdown</span>
```

Comment:
Is this correct, or should it be:

```suggestion
                <span class="caret sr-only">Toggle Dropdown</span>
```

Reply:
This is how the bootstrap docs show it. I will try your suggestion locally to see if anything changes.

Reply:
It should not be one line. This is what happens if you combine them: 
![image](https://github.com/nautobot/nautobot/assets/17092348/1a7029b9-a031-4f85-8efb-15055019ab9f)

Reply:
Cool :) Thanks for checking!